samples/extauthz/local-ext-authz.yaml

```diff
@@ -65,12 +65,11 @@ spec:
     spec:
       serviceAccountName: httpbin
       containers:
-      - image: docker.io/kong/httpbin
+      - image: docker.io/mccutchen/go-httpbin:v2.15.0
         imagePullPolicy: IfNotPresent
         name: httpbin
         ports:
-        - containerPort: 8001
-        command: ["gunicorn", "-b", "0.0.0.0:8001", "httpbin:app", "-k", "gevent"]
+        - containerPort: 8080
```

Comment:
The `containerPort` has changed in each file but the `targetPort` for each associated Service has not been updated. Does the `targetPort` need to be updated in each file?

Reply:
Thanks, Eoin. I have updated the service targetPort in the second commit.